### PR TITLE
 Use summary API when preview article

### PR DESCRIPTION
**Pull request by hueitan**
Phab. : https://phabricator.wikimedia.org/T242082

### CODE REVIEW

File: src/api/getArticleSumary.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { cachedFetch } from 'utils'
+
+export const getArticleSummary = (lang, title) => {
+  const url = `https://${lang}.wikipedia.org/api/rest_v1/page/summary/${encodeURIComponent(title)}`
```
Comment (hueitan) [inline src/api/getArticleSumary.js]: Let's do the abstraction when we modify `getArticle` and use new endpoint.

**Comment by hueitan** [comment]
@stephanebisson can you show me the example of `type: 'no-extract'`, I can't find one.

**Comment by stephanebisson** [comment]
On `en/Cat` the "carnivorous" link in the first paragraph returns "no-extract" for me. Let me know if you are seeing the same thing.

**Comment by hueitan** [comment]
Pending : Add loading experience when fetching the API.

**Comment by hueitan** [comment]
Capture

as agree with the design, we show the title when loading so user knows which article is being fetched.

**Comment by hueitan** [comment]
🐛 need to update the `text size adjustment` here .. argz,,,

**Comment by hueitan** [comment]
alright, this is ready to be reviewed again 🔥

**Comment by jpita** [comment]
ah damn, did `git push` instead of `git pull`.
can you check if I need to revert the commit @hueitan ?
it seems that it just brought the changes from master

**Comment by hueitan** [comment]
No, that’s fine, you don’t need to revert it.

**Comment by jpita** [comment]
> @jpita What's the status here, can we merge it soon?

sorry, totally forgot about this due to the task not being in QA

**Pull request merged**